test/soca/gw/config.base

```diff
@@ -95,6 +95,7 @@ export BASE_ENV="$HOMEgfs/env"
 export BASE_JOB="$HOMEgfs/jobs/rocoto"
 
 # EXPERIMENT specific environment parameters
+CDATE = ${PDY}${cyc}
```

Comment:
if you don't export it, it only works in this file

Reply:
Isn't that the idea?

Reply:
fair enough